examples/layers/ios/.gitignore

```diff
@@ -0,0 +1,37 @@
+.idea/
```

Comment:
cc @cbracken to sanity-check that this is valid for building these examples on iOS